go/apps/api/openapi/openapi-generated.yaml

```diff
@@ -2807,53 +2807,11 @@ components:
             type: object
             properties:
                 role:
-                    "$ref": "#/components/schemas/Role"
+                    "$ref": "#/components/schemas/role"
```

Comment:
unrelated, sorry for messing this PR up, this was required to make speakeasy happy though :)